rocky8/Dockerfile

```diff
@@ -26,16 +26,33 @@ RUN cd ~ && \
     cd ~ && \
     rm -rf ~/sqlite-autoconf*
 
+RUN cd ~ && \
+    yum -y install perl-IPC-Cmd perl-Pod-Html && \
+    wget -q https://github.com/openssl/openssl/releases/download/openssl-3.0.15/openssl-3.0.15.tar.gz && \
+    tar -xzf openssl-3.0.15.tar.gz && \
+    cd openssl-3.0.15 && \
+    ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl --libdir=lib no-ssl3 no-idea no-dtls no-srp no-comp shared && \
+    make && \
+    make install && \
+    cd ~ && \
+    rm -rf ~/openssl-3.0.15*
+
 # Compile newer version of python3
 RUN yum -y install openssl openssl-devel zlib-devel bzip2 bzip2-devel readline-devel tk-devel libffi-devel xz-devel && \
     cd ~ && \
     # Set up pyenv \
     git clone https://github.com/pyenv/pyenv.git ~/.pyenv && \
+    # setup python 3.10 \
     pyenv install 3.10 && \
+    # setup python 3.11 w/ openssl 3 \
+    PYTHON_CONFIGURE_OPTS="--with-openssl=/usr/local/ssl --with-openssl-rpath=auto --enable-shared --enable-optimizations" pyenv install 3.11 && \
     pyenv global 3.10 && \
     pip install --upgrade pip && \
     pip install --no-cache-dir py3createtorrent awscli && \
```

Comment:
so long as we're doing for one, should do for both pythons?  also in other files.